src/main/resources/hudson/plugin/versioncolumn/Messages.properties

```diff
@@ -1,3 +1,10 @@
-VersionMonitor.DisplayName=Version
-VersionMonitor.OfflineCause=This node is offline because it uses old slave.jar
-VersionMonitor.MarkedOffline=Making {0} offline temporarily due to the use of old slave.jar
+VersionMonitor.DisplayName=Remoting Version
+VersionMonitor.OfflineCause=This node is offline because it uses an old slave.jar
```

Comment:
I don't think we expose the term "remoting" anywhere (but perhaps the changelog) and we shouldn't start here.